src/main/java/com/gruzini/tennistico/messages/NotificationMessages.java

```diff
@@ -4,8 +4,8 @@
     public static final String MATCH_CREATED = "You have created an open match!";
     public static final String JOIN_REQUEST = "%s made a request to join your game!";
     public static final String JOIN_CONFIRMED = "%s has confirmed your join request. Match is set up. Good luck!";
-    public static final String SCORE_TO_SUBMIT = "Please submit score of your last game. Click and fill in";
-    public static final String SCORE_TO_CONFIRM = "%s have submitted the score of the match";
+    public static final String SCORE_TO_SUBMIT = "Your match vs %s took place recently. Please submit score. Click and fill in";
```

Comment:
Bardzo ważne pytanie: czy do Notification (albo nawet do samego NotificationDto) możnaby dodać PlayerDto. Chodzi o to, że inaczej nie da rady zrobić player-modala, a jest raczej ważne, chyba nawet najważniejsze w tym miejscu, żeby wiedzieć kogo dotyczy notyfikacja.

Reply:
do notification na pewno nie bedziemy dodawać dto bo to jest zapisywane w bazie. Troszkę nie rozumiem czemu ten komentarz jest w tej klasie, zakładam ze to przypadek.

No jeżeli chcemy aby na notyfikacji też była możliwość najechania na nazwisko i pokazania modala albo na score i pokazania danych meczu to może trzeba tam podlinkować wincyj danych... Tak czy inaczej dla mnie bardziej naturalne byłoby ładowanie tych danych podczas mapowania notyfikacji bazodanowej na dto. Tam IMO mamy dostępne wszystko co potrzebne. Za pomocą klas takich jak ten tutaj dodany parser możemy dopakować tego Dtosa.

Podaj proszę w takim razie co potrzebujesz - jakie dane to ja rozbuduje te parsery

Reply:
Nie do końca przypadek miejsce tego komentarza. Obecnie imię playera wchodzi jako element całego Stringa, więc na widoku będzie raczej niemożliwe wyłuskanie tylko samego imienia i tam podpięcie modala.

Ogólnie potrzebny mi jest PlayerDto oponenta w NotificationDto, ale trzeba by było może przemyśleć inny sposób wyświetlania treści samej notyfikacji. 

Też słuszna uwaga, że może będziemy chcieli później dodać modale również dla samego meczu (w sensie gdzie był rozegrany np.), ale na razie się nie rozpędzajmy, zakładam, że nie powinno być wielkim problemem dodanie tego później.

Reply:
Czyli chciałbyś zupełnie inaczej budować te message? No bo jeżeli chcemy wstawić linka to defaco treść messega trzeba zbudować na froncie. W jaki sposób wrzucić wtedy imię (lub score) do środka messega (nie na końcu nie na początku)? To bez sensu. W notification dto powinien faktycznie znalezc się przynajmniej id playera. Jakieś inne pomysły?

hmm po przemyśleniu uważam że może być warte świeczki przemyślenie sposobu dostarczania informacji na modala. Teraz każdy modal wymaga od nas władowanie wszystkich potrzebnych na nim informacji bezpośrednio do Dtosa który miał przenosić okrojone dane. Może ten modal przy otworzeniu by pytał jakieś api? Wtedy by wystarczyły IDki

Reply:
Jest to jakaś myśl, ale i tak niezależnie od tego jak chcielibyśmy podać tego Dtosa, to docelowo w notyfikacji powinniśmy kliknąć tylko imię gracza, żeby wyświetlił nam modala. Więc pewnie i tak wymagałoby to tworzenie treści dopiero na froncie.